frontend/packages/knative-plugin/src/actions/add-subscription.tsx

```diff
@@ -15,3 +21,9 @@ export const addSubscription = (model: K8sKind, source: K8sResourceKind): KebabO
     },
   };
 };
+
+export const addSubscriptionWithIcon = (source): KebabOption => {
+  const subscriptionMenu = addSubscription(EventingSubscriptionModel, source);
+  subscriptionMenu.icon = <StarIcon />;
```

Comment:
Remove the star icon. Fairly certain it was agreed now to have no icons.

cc @openshift/team-devconsole-ux

Reply:
Yes, i have removed it.